.circleci/scripts/binary_linux_test.sh

```diff
@@ -40,7 +40,7 @@ if [[ "$PACKAGE_TYPE" == conda ]]; then
     else
       cu_ver="${DESIRED_CUDA:2:2}.${DESIRED_CUDA:4}"
     fi
-    retry conda install -yq -c pytorch "cudatoolkit=\${cu_ver}"
+    retry conda install -yq -c nvidia -c pytorch "cudatoolkit=\${cu_ver}"
```

Comment:
This change is not needed now since the `cudatoolkit` package is available in the `defaults` channel.